cmd/vcert/config.go

```diff
@@ -41,6 +41,14 @@ func buildConfig(c *cli.Context, flags *commandFlags) (cfg vcert.Config, err err
 		var connectorType endpoint.ConnectorType
 		var baseURL string
 		var auth = &endpoint.Authentication{}
+
+		//case when access token can come from enviroment variable.
+		ttpTokenS := flags.tppToken
```

Comment:
Typo, variable name should be `tppTokenS`

Reply:
fixed on last commit